channels/rdpgfx/client/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ set(${MODULE_PREFIX}_SRCS
 	rdpgfx_main.h
 	rdpgfx_codec.c
 	rdpgfx_codec.h
-	rdpgfx_common.c
-	rdpgfx_common.h)
+	../rdpgfx_common.c
+	../rdpgfx_common.h)
```

Comment:
It could be better. 
However there's no precedent/sample to have static lib for something used by both channel server and channel client. But there's several existing similar 'common' stuff in other channels such as rdpei and rail.

Well, actually I don't know how to do it gracefully in current compile framework ^_^. Or we should make functions as static inline and put them in to xxxx_common.h? But anyway I think such work would be better resolved in a separate refactor PR for all of the xxx_common.c future.